pilot/pkg/serviceregistry/serviceentry/conversion.go

```diff
@@ -409,7 +409,7 @@ func convertWorkloadInstanceToServiceInstance(workloadInstance *model.WorkloadIn
 // Convenience function to convert a workloadEntry into a WorkloadInstance object encoding the endpoint (without service
 // port names) and the namespace - k8s will consume this workload instance when selecting workload entries
 func (s *Controller) convertWorkloadEntryToWorkloadInstance(cfg config.Config, clusterID cluster.ID) *model.WorkloadInstance {
-	we := ConvertWorkloadEntry(cfg)
```

Comment:
duplicate with

https://github.com/istio/istio/blob/f3f71b046f5cc79895103071a6af5f4ff4ca53d1/pilot/pkg/serviceregistry/serviceentry/controller.go#L225

Reply:
This seems easy to break in the future, can we make it clear it needs to be pre-converted?

Reply:
I haven't thought of a good way yet, so I reversed it first.